src/Webfactory/Slimdump/DumpTask.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Webfactory\Slimdump;
+
+use Symfony\Component\Console\Output\OutputInterface;
+use Webfactory\Slimdump\Config\Config;
+use Webfactory\Slimdump\Config\ConfigBuilder;
+use Webfactory\Slimdump\Database\Dumper;
+use Doctrine\DBAL\Connection;
+
+final class DumpTask
+{
+    /**
+     * DumpTask constructor.
+     * @param string $dsn
+     * @param string[] $configFiles
+     * @param OutputInterface $output
+     * @throws \Doctrine\DBAL\DBALException
+     */
+    public function __construct($dsn, $configFiles, OutputInterface $output)
+    {
+        $config = ConfigBuilder::createConfigurationFromConsecutiveFiles($configFiles);
+        $db = \Doctrine\DBAL\DriverManager::getConnection(
+            array('url' => $dsn, 'charset' => 'utf8', 'driverClass' => 'Doctrine\DBAL\Driver\PDOMySql\Driver')
+        );
+
+        $this->dump($config, $db, $output);
```

Comment:
I don't expect the constructor doing all the work eagerly. Consider removing this call, storing the variables in private fields and making the dump method public without arguments.